docker-compose.yml

```diff
@@ -9,37 +9,41 @@ services:
     image: postgres:14-alpine
     restart: always
     volumes:
-      - db-data:/var/lib/postgresql/data
+      - ./db-data:/var/lib/postgresql/data
     environment:
       - POSTGRES_PASSWORD=postgres
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
```

Comment:
This is very nice :) I'll try it out on my instance (together with ClickHouse container healthcheck)

Reply:
Hi, how's going with these healthchecks? 😎 Should we merge this PR?

Reply:
Sorry, I haven't tried them out yet.